api/src/migrations/0025_initial_gift_card.sql

```diff
@@ -20,4 +20,7 @@ CREATE TABLE IF NOT EXISTS `webshop_product_gift_card_mapping` (
     CONSTRAINT `gift_card_id_foreign` FOREIGN KEY (`gift_card_id`) REFERENCES `webshop_gift_card` (`id`),
     KEY `product_id_index` (`product_id`),
     CONSTRAINT `product_id_foreign` FOREIGN KEY (`product_id`) REFERENCES `webshop_products` (`id`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
\ No newline at end of file
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
+
+-- Make member_id nullable in Transactions, to enable Gift card purchasable without being a member
+ALTER TABLE `webshop_transactions` MODIFY COLUMN member_id int(10) unsigned NULL;
```

Comment:
If people have already applied migration 0025, then you should create a new migration, not append to an existing one.

Reply:
I appended to 0025 because 0025 has not been deployed yet. I checked with Daniel to ensure it was safe to append.